.renovaterc.json

```diff
@@ -32,7 +32,7 @@
   "semanticCommitType": "chore",
   "semanticCommitScope": null,
 
-  "separateMajorMinor": true,
+  "separateMajorMinor": false,
```

Comment:
What's the implication of this change?

Reply:
The current settings creates two PRs for a new LB release: one for major versions and one for other changes. As a result, each PR gets half of the newly released LB packages and both of them fails in many cases.